driver/cam_hal.c

```diff
@@ -207,6 +207,15 @@ static void cam_task(void *arg)
                         cam_obj->state = CAM_STATE_IDLE;
                     }
                     cnt++;
+                    // stop when too many DMA copies occur so the PSRAM
+                    // framebuffer slot doesn't overflow from runaway transfers
+                    if (cnt >= cam_obj->frame_copy_cnt) {
+                        ESP_LOGE(TAG, "DMA overflow");
+                        ll_cam_stop(cam_obj);
+                        cam_obj->state = CAM_STATE_IDLE;
+                        DBG_PIN_SET(0);
```

Comment:
please remove any DBG_PIN_* this is old and should not be used

Reply:
Okay, I will also remove the other occurrence there, if it's wrong anyway.